packages/components/pages/f-mfa/src/tenants/en-AU.js

```diff
@@ -1,4 +1,45 @@
-export default {
+const messages = {
     locale: 'en-AU',
-    text: 'I am a VMfa Component (AU)'
+    verificationPage: {
+        subTitle: "We've emailed your login code",
+        instructionsPrimaryText: "We've sent a verification code to {0} to help keep your account secure. Enter it below to log in.",
+        instructionsSecondaryText: 'Your code will be valid for 30 minutes.',
+        formField: {
+            labelText: 'Your verification code',
+            placeholderText: 'Enter code',
+            validationMessage: 'Please enter a valid code'
+        },
+        submitErrorText: 'Please check and try again',
+        primaryButtonText: 'Continue',
+        secondaryButtonText: 'Need help?'
+    },
+    helpInfo: {
+        title: 'Help',
+        subTitle: 'Need help?',
+        instructionsPrimaryText: "We've emailed a code to {0}. It will be valid for 30 minutes. If you're having trouble:",
+        instructionsPoint1Text: 'Wait a few minutes for the email to come through',
+        instructionsPoint2Text: 'Check your spam folder',
+        instructionsPoint3Text: 'Try logging in another way',
+        instructionsPoint4Text: 'See our {0} if you still need help',
+        instructionsHelpLink: 'https://www.menulog.com.au/help/article/5448178892303/why-am-i-sometimes-asked-to-enter-a-verification-code-when-using-menulog',
```

Comment:
This link loads a blank page. Same with the other?

Reply:
🦅👁️
 The pages haven't been published yet, but they will be closer to release. The URLs should be correct though.